### ISSUE TITLE
/auth/edit broken when language set to Finnish

### ISSUE THREAD

**Issue by SHeija** [tootsuite/mastodon#8887]
As a user, when mastodon's UI language is set to Finnish, security page in user preferences (/auth/edit) returns an error page.

All other languages work fine, and browser language has no effect. I have tested this on both firefox and chrome, on multiple servers running 2.5.0.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
-  [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by jaranta** [comment]
Just confirmed this on 2.5.1.

**Comment by qguv** [comment]
The issue is a bad Finnish localization string. Found another bad one in the Welsh localization with:

```
$ grep -o -Ee '%{[^}]+}' *.yml | cut -d':' -f2 | sort | uniq -c | sort -r
1760 %{name}
1057 %{target}
1017 %{count}
[...]
  28 %{sign_up_path}
  22 %{value}
   3 %{limit}
   1 %{taget}
   1 %{selain}
```

`taget` is "browser" in Finnish, and `selain` is a typo in the Welsh locale, cy.yml. Both are fixed by #8994.
